packages/angular/projects/clr-angular/src/data/datagrid/datagrid-row-detail.ts

```diff
@@ -57,4 +64,20 @@ export class ClrDatagridRowDetail implements AfterContentInit, OnDestroy {
   ngOnDestroy() {
     this.subscriptions.forEach(sub => sub.unsubscribe());
   }
+
+  @Input('clrRowDetailBeginningAriaText') ariaTextBeginningOfExpandableContent: string;
+  public get beginningOfExpandableContent() {
+    return (
+      this.ariaTextBeginningOfExpandableContent ||
```

Comment:
nit: these common string property names should be aligned with their corresponding input property names too.